source/ODApplication.cpp

```diff
@@ -87,8 +92,6 @@ void ODApplication::startServer()
 void ODApplication::startClient()
 {
     ResourceManager& resMgr = ResourceManager::getSingleton();
-    LogManagerOgre logManager(resMgr.getLogFile());
-    logManager.setLogDetail(LogMessageLevel::NORMAL);
```

Comment:
I will check but if the log manager was launched here too, I think that's because it is not launched the same way in client (normal way) or in server (using a command line, allows to start a server where the clients will connect to play the game)